website/docs/releases/2024/v2024.next.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Release 2024.next
+slug: "/releases/2024.next"
+---
+
+:::::note
+2024.next has not been released yet! We're publishing these release notes as a preview of what's to come, and for our awesome beta testers trying out release candidates.
+
+To try out the release candidate, replace your Docker image tag with the latest release candidate number, such as 2024.next.0-rc1. You can find the latest one in [the latest releases on GitHub](https://github.com/goauthentik/authentik/releases). If you don't find any, it means we haven't released one yet.
+:::::
+
+## Breaking changes
+
+### PostgreSQL minimum supported version upgrade
+
+authentik now requires PostgreSQL version 14 or later. We recommend upgrading to the latest version if you are running an older version.
+
+The provided Helm chart defaults to PostgreSQL 15. If you are using the Helm chart with the default values, no action is required.
+
+The provided docker-compose file was updated with PostgreSQL 16. You can follow the procedure [here](../../troubleshooting/postgres/upgrade_docker.md) to upgrade.
```

Comment:
We should consider saying "Compose file" because Docker has deprecated `docker-compose`. 

In general, when referring to the product called Docker Compose, we capitalize and do not use a hyphen. When referring to a command, using courier font, then use lower-case and no hyphen `docker compose`.  For the file, when you are just referring to it without naming it, let's go with "Compose file".  If naming it, then use code format and use the exact name of file (`docker-compose.yml`).